.github/workflows/deploy.yml

```diff
@@ -27,10 +27,11 @@ jobs:
       - name: Install modules
         run: npm ci
 
-      - name: Run JSDoc
+      - name: Generate JSDoc files and copy to dist
         run: |
           rm -rf ./source/docs
           npm run generate-docs
+          npm run copy-docs
```

Comment:
Maybe we can copy the docs over after we build the code? In this way we can keep `npm run clean` in the build script